packages/openneuro-components/src/facets/FacetListWrap.tsx

```diff
@@ -23,49 +29,61 @@ export const FacetListWrap = ({
   items,
   selected,
   setSelected,
+  startOpen,
+  label,
+  accordionStyle,
+  dropdown,
 }: FacetListWrapProps) => {
   const setSetter = (e, item) => {
     e.stopPropagation() // Stop this click event to trigger click on parent onClick()
     setSelected(item)
   }
   return (
-    <div className="facet-list">
-      <ul className="level-1">
-        {items.map((item, index) => (
-          <li
-            key={index}
-            onClick={e => setSetter(e, item)}
-            className={
-              selected && selected.value == item.value
-                ? 'selected-facet facet'
-                : 'facet'
-            }>
-            <span className="label">
-              {item.label}
-              {item.count && <span>({item.count})</span>}
-            </span>
-            {item.children && (
-              <ul className="level-2">
-                {item.children.map((item, index) => (
-                  <li
-                    key={index}
-                    onClick={e => setSetter(e, item)}
-                    className={
-                      selected && selected.value == item.value
-                        ? 'selected-facet facet'
-                        : 'facet'
-                    }>
-                    <span className="label">
-                      {item.label}
-                      {item.count && <span>({item.count})</span>}
-                    </span>
-                  </li>
-                ))}
-              </ul>
-            )}
-          </li>
-        ))}
-      </ul>
-    </div>
+    <AccordionWrap className="facet-accordion">
+      <AccordionTab
+        accordionStyle={accordionStyle}
+        label={label}
+        startOpen={startOpen}
+        dropdown={dropdown}>
+        <div className="facet-list">
+          <ul className="level-1">
+            {items.map((item, index) => (
+              <li
+                key={index}
+                onClick={e => setSetter(e, item)}
+                className={
+                  selected && selected.value == item.value
+                    ? 'selected-facet facet'
```

Comment:
I'm not seeing the color change, but the class is getting updated. Is that expected for the storybook component?

Reply:
it was subtle because it was using the primary color. I adjusted it to use the header primary color.